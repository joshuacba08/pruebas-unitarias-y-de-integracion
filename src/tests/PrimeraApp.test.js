// import { render } from "@testing-library/react";
import PrimeraApp from "../PrimeraApp";
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';

describe('Pruebas en <PrimeraApp />', () => {
    
    // test('Debe mostrar el mensaje "Hola, soy Josué"', () => {
    //     const saludo = "Hola, soy Josué";
    //     const { getByText } = render( <PrimeraApp saludo ={ saludo }/>)
    //     expect( getByText(saludo) ).toBeInTheDocument(); 
    // })
    test('Debe de mostrar el compotente <PrimeraAPP /> correctamente', () => {
        
        const saludo = "Hola, soy Josué"
        const wrapper = shallow( <PrimeraApp saludo ={ saludo } /> );

        expect( wrapper ).toMatchSnapshot();
    })
    


})
