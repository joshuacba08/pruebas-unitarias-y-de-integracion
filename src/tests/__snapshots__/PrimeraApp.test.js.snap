// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> Debe de mostrar el compotente <PrimeraAPP /> correctamente 1`] = `ShallowWrapper {}`;
